dracut/30ignition/ignition-generator

```diff
@@ -53,8 +53,6 @@ Requires=dev-disk-by\x2dlabel-boot.device
 After=dev-disk-by\x2dlabel-boot.device
 EOF
     fi
-
-    add_requires coreos-teardown-initramfs-network.service ignition-complete.target
```

Comment:
Hmm, I wonder if there was a reason we enabled it via the generator before vs statically. Ahhh, probably because it predates the backporting of ignition-complete.target.